stablehlo/reference/Ops.cpp

```diff
@@ -102,6 +102,94 @@ Tensor evalSliceOp(const Tensor &operand, const Sizes &startIndices,
                      inferredTypes[0].cast<ShapedType>());
 }
 
+template <typename T>
+DenseIntElementsAttr getDenseIntElementsAttr(
```

Comment:
I do not see the point of this generic function. 
We are using this function to create a `DenseIntElementsAttr` to be passed on to `inferReduceOp`. Are we using this function is some other places as well?